.github/workflows/README.md

```diff
@@ -0,0 +1,32 @@
+# Use workflows in forked repositories
```

Comment:
It makes no sense to me to have and maintain this in this repo, if you fork it and need to adjust, adjust the workflows as well.